conf/deployment/vsphere/upi_1az_rhcos_vsan_compact_mode_3m_0w.yaml

```diff
@@ -0,0 +1,17 @@
+---
+# This config is suppose to work on most of DCs we have.
+DEPLOYMENT:
+  allow_lower_instance_requirements: false
+ENV_DATA:
+  platform: 'vsphere'
+  deployment_type: 'upi'
+  worker_replicas: 0
+  master_replicas: 3
+  worker_num_cpus: '16'
+  master_num_cpus: '4'
+  master_memory: '16384'
```

Comment:
with 4 CPU and 16gb ram, i don't think that will be sufficient for running OCS on them

Reply:
I've updated the values for master nodes to reflect what our worker nodes are using in standard deployments and removed the values for workers as we don't have any worker nodes anyways. @petr-balogh FYI.